src/browser/tab/mod.rs

```diff
@@ -663,6 +663,28 @@ impl<'a> Tab {
             .call_method(protocol::debugger::methods::GetScriptSource { script_id })?
             .script_source)
     }
+
+    /// Executes javascript
```

Comment:
The first sentence from the original docs is "Evaluates expression on global object.", which is a much better description thatn "Executes javascript".

Reply:
Thanks.. Made the changes